src/pages/Nav/Nav.scss

```
@@ -0,0 +1,298 @@
+.signBar {
```

```suggestion
@import '../../styles/mixin';

.signBar {
```

import 해주지 않으면 include를 사용할 수 없습니다!